app_test.go

```diff
@@ -477,3 +478,260 @@ func TestGetAppNoTarget(t *testing.T) {
 		t.Fatal("unexpected <nil> error")
 	}
 }
+
+func TestSetConfig(t *testing.T) {
+	fakeServer := newFakeServer(t)
+	defer fakeServer.stop()
+	fakeServer.prepareResponse(preparedResponse{
+		code:    http.StatusOK,
+		method:  "POST",
+		path:    "/apps/proj1-prod/env",
+		payload: []byte("{}"),
+	})
```

Comment:
so an integration test ? I like the idea, and I agree it would be more practical (: